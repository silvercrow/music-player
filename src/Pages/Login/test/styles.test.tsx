import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import {
  Page,
  Container,
  Wrapper,
  LoginBox,
  Title,
  LoginButtonContainer,
  GoogleButton,
  Icon,
  ButtonText,
} from '../styles';

test('Page', () => {
  const tree = renderer.create(<Page />).toJSON();
  expect(tree).toMatchSnapshot();
});

test('Container', () => {
  const tree = renderer.create(<Container />).toJSON();
  expect(tree).toMatchSnapshot();
});

test('Wrapper', () => {
  const tree = renderer.create(<Wrapper />).toJSON();
  expect(tree).toMatchSnapshot();
});

test('LoginBox', () => {
  const tree = renderer.create(<LoginBox />).toJSON();
  expect(tree).toMatchSnapshot();
});

test('Title', () => {
  const tree = renderer.create(<Title />).toJSON();
  expect(tree).toMatchSnapshot();
});

test('LoginButtonContainer', () => {
  const tree = renderer.create(<LoginButtonContainer />).toJSON();
  expect(tree).toMatchSnapshot();
});

test('GoogleButton', () => {
  const tree = renderer.create(<GoogleButton />).toJSON();
  expect(tree).toMatchSnapshot();
});

test('Icon', () => {
  const tree = renderer.create(<Icon />).toJSON();
  expect(tree).toMatchSnapshot();
});

test('ButtonText', () => {
  const tree = renderer.create(<ButtonText />).toJSON();
  expect(tree).toMatchSnapshot();
});
