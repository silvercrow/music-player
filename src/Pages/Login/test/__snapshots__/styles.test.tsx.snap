// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonText 1`] = `
.c0 {
  display: inline-block;
  vertical-align: middle;
  padding-left: 42px;
  padding-right: 42px;
  font-size: 14px;
  font-weight: bold;
  font-family: 'Roboto',sans-serif;
}

<span
  className="c0"
/>
`;

exports[`Container 1`] = `
.c0 {
  width: 100%;
  min-height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -moz-box;
  display: -ms-flexbox;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding: 15px;
  background: #9053c7;
  background-color: #f39f86;
  background-image: linear-gradient(315deg,#f39f86 0%,#f9d976 74%);
}

<div
  className="c0"
/>
`;

exports[`GoogleButton 1`] = `
.c0 {
  display: inline-block;
  background: white;
  color: #444;
  border-radius: 5px;
  border: thin solid #888;
  box-shadow: 1px 1px 1px grey;
  white-space: nowrap;
}

.c0:hover {
  cursor: pointer;
}

<div
  className="c0"
/>
`;

exports[`Icon 1`] = `
.c0 {
  background: url('https://upload.wikimedia.org/wikipedia/commons/thumb/5/53/Google_%22G%22_Logo.svg/512px-Google_%22G%22_Logo.svg.png') transparent 5px 50% no-repeat;
  display: inline-block;
  vertical-align: middle;
  width: 34px;
  height: 34px;
  background-size: cover;
  padding: 7px;
}

<span
  className="c0"
/>
`;

exports[`LoginBox 1`] = `
<div
  className=""
/>
`;

exports[`LoginButtonContainer 1`] = `
.c0 {
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -moz-box;
  display: -ms-flexbox;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  padding-top: 20px;
}

<div
  className="c0"
/>
`;

exports[`Page 1`] = `
.c0 {
  width: 100%;
  margin: 0 auto;
}

<div
  className="c0"
/>
`;

exports[`Title 1`] = `
.c0 {
  font-size: 24px;
  color: #333333;
  line-height: 1.2;
  text-align: center;
  width: 100%;
  display: block;
  padding-bottom: 12px;
}

<span
  className="c0"
/>
`;

exports[`Wrapper 1`] = `
.c0 {
  background: #fff;
  border-radius: 10px;
  overflow: hidden;
  display: -webkit-box;
  display: -webkit-flex;
  display: -moz-box;
  display: -ms-flexbox;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  padding: 30px 45px 33px 60px;
}

<div
  className="c0"
/>
`;
